ui/raidboss/data/03-hw/dungeon/gubal_library_hard.txt

```diff
@@ -0,0 +1,157 @@
+1108.4 "Seal Of Night And Day" sync /:Liquid Flame:1949:/ window 10,10
+1112.5 "Searing Wind" sync /:Liquid Flame:1948:/
+1116.4 "Searing Wind" sync /:Liquid Flame:1948:/
+1126.5 "Seal Of Night And Day" sync /:Liquid Flame:1949:/ window 10,10
+1130.6 "Searing Wind" sync /:Liquid Flame:1948:/
+1134.7 "Searing Wind" sync /:Liquid Flame:1948:/
+1138.9 "Searing Wind" sync /:Liquid Flame:1948:/
+1149.1 "Form Shift"
+
+# Tornado Form
+1200.0 "--sync--" sync /00:....:The liquid flame gains the effect of Anemomorph/ window 200,0
+1204.5 "Bibliocide" sync /:Liquid Flame:1945:/ window 10,10
+1215.4 "Magnetism/Repel?" sync /:Liquid Flame:194[CD]:/
+1226.9 "Bibliocide" sync /:Liquid Flame:1945:/
+1237.8 "Magnetism/Repel" sync /:Liquid Flame:194[CD]:/
+1246.1 "Form Shift"
+
+# Human Form
+1300.0 "--sync--" sync /00:....:The liquid flame gains the effect of Anthropomorph/ window 300,10
+1307.5 "Searing Wind" sync /:Liquid Flame:1944:/ window 10,20
+1314.7 "Bibliocide" sync /:Liquid Flame:1945:/
+1326.0 "Sea Of Flames x3"
+1331.3 "Slosh" sync /:Liquid Flame:1947:/ window 10,10
+1335.5 "Searing Wind" sync /:Liquid Flame:1944:/
+1343.8 "Bibliocide" sync /:Liquid Flame:1945:/
+
+1355.2 "Searing Wind" sync /:Liquid Flame:1944:/ window 10,20
+1362.4 "Bibliocide" sync /:Liquid Flame:1945:/
+1373.8 "Sea Of Flames x3"
+1379.0 "Slosh" sync /:Liquid Flame:1947:/ window 10,10
+1383.2 "Searing Wind" sync /:Liquid Flame:1944:/
+1391.5 "Bibliocide" sync /:Liquid Flame:1945:/ jump 1343.8
+
+1402.9 "Searing Wind"
+1410.1 "Bibliocide"
+1421.4 "Sea Of Flames x3"
+1426.7 "Slosh"
+
+
+# Strix
+# -ii 194F 1950 1958 195B 1969
+
+2000.0 "--sync--" sync /The Rare Tomes Room will be sealed off/ window 2000,5
+2009.2 "Check Out" sync /:Strix:194E:/
+2017.5 "Properties Of Darkness (buster)" sync /:Strix:1954:/
+2029.6 "Properties Of Quakes" sync /:Strix:1956:/
+2033.8 "Properties Of Darkness (buster)" sync /:Strix:1954:/
+2040.1 "Check Out" sync /:Strix:194E:/
+2048.4 "Properties Of Darkness (buster)" sync /:Strix:1954:/
+2056.7 "Properties Of Darkness II" sync /:Strix:1955:/
+2069.9 "Properties Of Tornados" sync /:Strix:1957:/
+2074.2 "Properties Of Darkness (buster)" sync /:Strix:1954:/
+2082.4 "Properties Of Imps" sync /:Strix:1959:/
+2085.5 "Properties Of Darkness" sync /:Strix:1954:/
+2094.7 "Properties Of Thunder III" sync /:Strix:195A:/
+
+# Intermission
+2109.2 "Check Out" sync /:Strix:194E:/
+2125.6 "Properties Of Darkness (buster)" sync /:Strix:1954:/
+2126.1 "--sync--" sync /:Meteor:1A6A:/
+2126.7 "Meteor Impact" sync /:Behemoth Ward:195E:/
+2133.8 "Properties Of Darkness (buster)" sync /:Strix:1954:/
+2135.1 "--sync--" sync /:Meteor:1A6A:/
+2135.6 "Meteor Impact" sync /:Behemoth Ward:195E:/
+2143.7 "Ecliptic Meteor" sync /:Behemoth Ward:195D:/
+2144.7 "--sync--" sync /:Behemoth Ward:195C:/
+
+2155.0 "Check Out" sync /:Strix:194E:/ window 20,20
+2172.3 "Quakes/Tornados" sync /:Strix:195[67]:/
```

Comment:
I don't think I ever knew this was random!

Reply:
I didn't either, I had always assumed it alternated. We learn SO much doing this.